webapp/platform/types/src/config.ts

```diff
@@ -213,6 +213,7 @@ export type ClientConfig = {
     UniqueEmojiReactionLimitPerPost: string;
     UsersStatusAndProfileFetchingPollIntervalMilliseconds: string;
     YoutubeReferrerPolicy: 'true' | 'false';
+    ScheduledPosts: string;
```

Comment:
q: why this type needs to be a string?

Reply:
Thats just the way we return boolean values in client config. I don't know whats the reason, but yeah, every boolean setting is passed as string

![95vrug](https://github.com/user-attachments/assets/c6ba05e1-fc32-4aab-9cb6-28bc02ceb2df)